edgecast/waf/custom_rule.go

```diff
@@ -10,6 +10,15 @@ import (
 	This file contains operations and types specific to WAF Custom Rule Sets.
 */
 
+// DeleteCustomRuleResponse contains the response from the WAF API when deleting a new rule
+type DeleteCustomRuleResponse struct {
```

Comment:
This may no longer be needed if the base struct is implemented as discussed above

Reply:
Updated to include WAF response struct + one additional property needed